packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -115,6 +115,7 @@ export interface FileStateProps {
   error: boolean;
   paused: boolean;
   errorMessage: string;
+  percentage?: number;
```

Comment:
We have percentage on `FileStatus` at line 105 above, is this one needed?

Reply:
I think we might need to clean up the types a bit, `FileStateProps` is used on the FileState component, so I could remove line 105 above since it extends `FileStateProps`